packages/docs/package.json

```diff
@@ -1,5 +1,11 @@
 {
   "name": "@lest/docs",
   "private": true,
-  "description": "API documentation in JSON form."
+  "description": "API documentation in JSON form.",
+  "main": "dist/src/index.js",
+  "type": "commonjs",
+  "scripts": {
```

Comment:
Could you also add a `format` script which runs prettier on the JSON and TS?

In the future we probably also want to assert that all json docs are exported from the top level, but that would need Jest set up and a test written to detect all JSON docs.